app/javascript/app/components/header/header-styles.scss

```diff
@@ -1,29 +1,8 @@
-.header {
-  margin: 10px 0 0;
-  padding: 10px 0;
-  border-bottom: 1px solid #eee;
-}
-
-.link {
-  display: inline-block;
-  margin: 0 10px 0 0;
-  text-decoration: none;
+@import '../../styles/settings';
 
-  &Active {
-    text-decoration: underline;
-  }
-
-  &:hover {
-    color: #ccc;
-  }
-}
-
-.nav {
-  padding: 0;
-  list-style: none;
-  display: flex;
-}
-
-.navItem {
-  margin-right: 10px;
+.header {
+  background-color: $theme-color;
+  min-height: 300px;
```

Comment:
I think I'd move to a prop in the component to define small or large because we have differents heights:
https://app.zeplin.io/project/598b1dc89689e51e1cd1b5ce/screen/598b1e52169b9d278d8f8091 (small)
https://app.zeplin.io/project/598b1dc89689e51e1cd1b5ce/screen/598b1e4885316be317c8f2ab (large)